cookbook: make "interpretation of results" more concise (#175)
RMI-PACTA/pacta.loanbook: vignettes/articles/cookbook_interpretation.Rmd

### Trajectory Plot

> The production volume trajectory metric evaluates a portfolio’s alignment with climate scenarios by comparing projected production volumes---based on companies' five-year capital expenditure plans---to scenario targets. It applies to the fossil fuels, power, and automotive sectors, where production changes occur either through shifts between technologies (e.g., internal combustion engines to electric vehicles) or overall expansion/contraction (e.g., adding new coal-fired power plants). The trajectory reflects these changes at the technology level for individual clients and is compared to climate scenario trends.

Reviewer: ```suggestion
The production volume trajectory metric evaluates a portfolio’s alignment with climate scenarios by comparing projected production volumes---based on companies' five-year capital expenditure plans---to scenario targets. It applies to the fossil fuels, power, and automotive sectors, where production changes occur either through shifts between technologies (e.g., internal combustion engines to electric vehicles) or overall expansion/contraction (e.g., closing down coal mines). The trajectory reflects these changes at the technology level for individual clients and is compared to climate scenario trends.
```
I think it may be better to use an upstream cola or O&G example here, because these sectors are the ones, where PACTA does not really say anything about technology mix. Sector pathways are clearly focused on phasing down the production.

Author: Resolved by [cf431fb](https://github.com/RMI-PACTA/pacta.loanbook/pull/175/commits/cf431fbd9cba551ce0e5a32b38def3afcfe66d1c)

#### Explanation of Output

> The trajectory plot displays a portfolio’s production volume for a specific technology or fuel type, tracking changes from the start year to five years into the future (**solid black line**). This projection is compared to the benchmark region’s production trajectory (**dashed black line**). The plot’s background is color-coded: green indicates alignment with ambitious climate scenarios, red signifies misalignment, and shades of yellow for in-between. For low-carbon technologies, values above the most ambitious scenario appear green, and those below the least ambitious scenario appear red; the opposite applies to high-carbon technologies. Typically, the plot includes multiple scenarios but can also compare the portfolio against a single scenario.

Reviewer: Since this plot is a frequent subject of confused questions, I thought it may be better to clarify that scenario pathways are the lines delineating the colors. I am also realizing that we wanted to clarify what alignment in this plot means in terms of temperature outcome (another recurring question). I don't think we do that at the moment.

#### Interpretation & Intuition

A portfolio is aligned with a climate scenario if its production trajectory (**solid black line**) follows the scenario line. Low-carbon technologies generally require increasing production under ambitious scenarios, whereas high-carbon technologies must decline or phase-out. If a portfolio scales up low-carbon production faster than the scenario, it supports the transition; if it lags, it slows progress.

> More ambitious scenarios assume stronger shifts toward low-carbon solutions, leading to lower global heating outcomes. Thus, the degree of ambition in a portfolio’s production trajectory directly correlates to its alignment with climate targets.

Reviewer: This corresponds to my previous comment. Maybe we should clarify that the lines refer to scenarios, which model a temperature outcome with some uncertainty. And that (1) any individual portfolio/company cannot really be said to be on track with a global temperature outcome in a a vacuum and (2) the areas outside of the scenario lines cannot be translated to any temperature outcome based on the plot.

Author: Since this is sort of "new content" I've flagged it as a new issue #178 
Something we can consider elaborating on between the FAQ call and the final "release", but I don't want to spend any more time wordsmithing this before we send it to IFC.

## Sectoral Decarbonization (SDA) Target

The sectoral decarbonization approach (SDA) calculates PACTA alignment metrics for aviation, cement, and steel—sectors. These sectors are considered "hard-to-abate" due to the lack of market-ready low-carbon alternatives. As a result, there are no clear production-pathways at the level of technology or fuel-type. Unlike the market share approach, which tracks technology shifts, the SDA evaluates sector-level emission intensities (e.g., tonnes of CO~2~ per tonne of cement or steel produced), providing a comparable measure across all production methods.

> The SDA determines the required carbon intensity for a sector---as specified by a climate scenario---by dividing a sector-specific carbon budget by expected production volumes. The starting emission intensity is calculated as current GHG emissions divided by total sector output. To align with the scenario, companies must follow an individual decarbonization trajectory that starts at their current emission intensity and converges with a common sector-wide target by the scenario’s endpoint (e.g., 2050). Companies with lower initial intensities require less reduction than those above the sector average. If all companies in a sector decarbonize their production in a way that follows the average emission intensity of the sector, then the scenario targets will be reached in aggregate. 

Reviewer: ```suggestion
The SDA determines the required carbon intensity for a sector---as specified by a climate scenario---by dividing a sector-specific carbon budget by expected production volumes. The starting emission intensity is calculated as current GHG emissions divided by total sector output. To align with the scenario, companies must follow an individual decarbonization trajectory that starts at their current emission intensity and converges with a common sector-wide target by the scenario’s endpoint (e.g., 2050). Companies with lower initial intensities require less reduction than those above the sector average. If all companies in a sector decarbonize their production in a way that follows the average emission intensity of the sector, then the scenario targets will be reached in aggregate. 
```
The last sentence sounds ambiguous. It sounds as if we are now suggesting they track the sector average instead of their individual target pathways. But not sure how to improve this formulation

Author: Maybe something like:

> If all companies in a sector decarbonize their production in a way that coverges their average emission intensity to the sectoral target, then the scenario target will be reached in aggregate.

Reviewer: sounds good!

> The SDA determines the required carbon intensity for a sector---as specified by a climate scenario---by dividing a sector-specific carbon budget by expected production volumes. The starting emission intensity is calculated as current GHG emissions divided by total sector output. To align with the scenario, companies must follow an individual decarbonization trajectory that starts at their current emission intensity and converges with a common sector-wide target by the scenario’s endpoint (e.g., 2050). Companies with lower initial intensities require less reduction than those above the sector average. If all companies in a sector decarbonize their production in a way that follows the average emission intensity of the sector, then the scenario targets will be reached in aggregate. 

Author: ```suggestion
The SDA determines the required carbon intensity for a sector---as specified by a climate scenario---by dividing a sector-specific carbon budget by expected production volumes. The starting emission intensity is calculated as current GHG emissions divided by total sector output. To align with the scenario, companies must follow an individual decarbonization trajectory that starts at their current emission intensity and converges with a common sector-wide target by the scenario’s endpoint (e.g., 2050). Companies with lower initial intensities require less reduction than those above the sector average. If all companies in a sector decarbonize their production in such a way that their average emission intensity converges to the sectoral target, then the scenario target will be reached in aggregate.
```

For more details on the Sectoral Decarbonization (SDA) metric and its calculation, see the [Metrics appendix](cookbook_metrics.html#sectoral-decarbonization-approach).

### `target_sda()` function

The `target_sda()` function calculates the Sectoral Decarbonization Target metric, producing a table that includes the calculated targets at the portfolio or company level, depending on the options selected.

#### Explanation of Output

The primary variable of interest in the output is `emission_factor_value`, which represents an emission intensity depending on the possible values of the `emission_factor_metric` column`:

- `"projected"`: The portfolio’s physical emission intensity based on ABCD data for the sector, year, region, and scenario.

> - `"target_*"` (e.g., `"target_cps"`): The scenario-aligned target emission intensity, starting from the portfolio’s initial intensity and converging with the sector-level scenario target by the final year.

Reviewer: ```suggestion
- `"target_*"` (e.g., `"target_cps"`): The scenario-aligned individual target emission intensity, starting from the portfolio’s initial intensity and converging with the sector-level scenario target by the final year.
```

- `"corporate_economy"`: The weighted average emission intensity of the entire sector, serving as a benchmark for comparison.
- `"adjusted_*"`: The scenario trajectory for emission intensity, adjusted for differences between ABCD sector totals and scenario data.

When `by_company = TRUE`, the output is specific to the company listed in the `name_abcd` column.

For detailed information about the table output of the `target_sda()` function and its columns, see the [data dictionary](data_outputs.html#target_sda-output).

### Emission Intensity Plot

> The emission intensity metric measures the average CO~2~ intensity of a portfolio in the `r cli::format_inline("{unique(co2_intensity_scenario_demo$sector)} sector{?s}")`, expressed as CO~2~ per unit of economic output (e.g., CO2 per ton of steel produced). This five-year forward-looking trajectory is compared to an emission intensity reference trend based on a climate transition scenario.

Reviewer: ```suggestion
The emission intensity metric measures the average CO~2~ intensity of a portfolio in the aviation, cement, and steel sectors, expressed as CO~2~ per unit of economic output (e.g., CO2 per ton of steel produced). This five-year forward-looking trajectory is compared to an emission intensity reference trend based on a climate transition scenario.
```
listing the sectors based on the demo data is not a good idea, because the aviation sector is not included in that data set. I would suggest we hard-code the sectors for now,  which is not ideal, but correct

> This metric is used for sectors without clear technology pathways or scalable zero-carbon alternatives (namely, `r cli::format_inline("{unique(co2_intensity_scenario_demo$sector)}")`). Since technology mix and production volume trajectory metrics are not applicable, emission intensity serves as the primary measure for assessing alignment and guiding capital toward emissions reductions in these sectors.

Reviewer: ```suggestion
This metric is used for sectors without clear technology pathways or scalable zero-carbon alternatives (namely, aviation, cement and steel). Since technology mix and production volume trajectory metrics are not applicable, emission intensity serves as the primary measure for assessing alignment and guiding capital toward emissions reductions in these sectors.
```
same issue as above
